Automattic/jetpack
# VideoPress: move source-client code from plugin to package

This PR moves the source code of the admin page of the VideoPress feature, from the plugin to the package. It requires move the compiling process, too.

Fixes https://github.com/Automattic/jetpack/issues/25684

#### Changes proposed in this Pull Request:

* VideoPress: move source-client code from plugin to package
* VideoPress: implement compiling process for the VideoPress admin page

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Test the VideoPress compiling process

```
jetpack build packages/videopress
```

* Confirm the built folder

```cli
ll build
```

```
drwxr-xr-x   8 retrofox  staff   256B Aug 17 15:30 admin
drwxr-xr-x  14 retrofox  staff   448B Aug 17 15:30 block-editor
drwxr-xr-x   5 retrofox  staff   160B Aug 17 15:30 lib
```

the `admin` folder contains built files to be served in the cleint, wp-admin context


* Test with a Jetpack site
* Activate VideoPress plugin
* Go to wp-admin
* Go to VideoPress plugin
* Confirm the page looks as expected
* Take a look at the built files
<img width="1728" alt="Screen Shot 2022-08-17 at 16 22 25" src="https://user-images.githubusercontent.com/77539/185225189-ee0fe86b-cce3-4117-9c86-7281b0c5691c.png">
